src/applications/mhv/shared/util/helpers.js

```diff
@@ -0,0 +1,42 @@
+import { formatDateLong } from '@department-of-veterans-affairs/platform-utilities/exports';
+import moment from 'moment';
+
+/**
+ * @param {Object} nameObject {first, middle, last, suffix}
+ * @returns {String} formatted timestamp
+ */
+export const formatName = ({ first, middle, last, suffix }) => {
+  let name = `${last}, ${first}`;
+  if (middle) name += ` ${middle}`;
+  if (suffix) name += `, ${suffix}`;
+  return name;
+};
+
+/**
+ * @param {Object} user user profile object from redux store (state.user.profile)
+ * @param {Object} title title of the doc (displayed at the top of the doc)
+ * @param {Object} subject subject of the doc (metadata)
+ * @param {Object} preface preface below the title (displayed below the title)
+ * @returns {String} formatted timestamp
+ */
+export const generatePdfScaffold = (user, title, subject, preface) => {
+  const name = formatName(user.userFullName);
+  const dob = formatDateLong(user.dob);
+  return {
+    headerLeft: name,
+    headerRight: `Date of birth: ${dob}`,
+    headerBanner: [
+      {
+        text:
```

Comment:
Can you add the bold formatting in the VCL banner? Unless it's been removed from the comps and I missed it.

Reply:
Does PDFkit accept inline formatting?

Reply:
Oh yes, forgot. Thanks!

Reply:
A very clever way to implement bold text in the header banner!